pkg/storj/object.go

```diff
@@ -97,4 +97,5 @@ type Segment struct {
 type Piece struct {
 	Number   byte
 	Location NodeID
+	Online   bool
```

Comment:
This does not make sense. `storj` package describes the data model how it's stored and communicated not current status of things. This PR should not change `storj` package.

Reply:
removed the changes.